pkgs/development/python-modules/manuf/internal_db_update_nix.patch

```diff
@@ -0,0 +1,51 @@
+diff --git a/manuf/manuf.py b/manuf/manuf.py
+index e5e9193..0ac9296 100755
+--- a/manuf/manuf.py
++++ b/manuf/manuf.py
+@@ -61,12 +61,19 @@ class MacParser(object):
+         IOError: If manuf file could not be found.
+ 
+     """
+-    MANUF_URL = "https://gitlab.com/wireshark/wireshark/raw/master/manuf"
++    # https://github.com/coolbho3k/manuf/issues/34
++    MANUF_URL = "https://www.wireshark.org/download/automated/data/manuf"
```

Comment:
Is this related to the db update?
If not, it would be easier to maintain if we split it into separate patches.

Reply:
Done